src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/EnumBuilder.cs

```diff
@@ -323,11 +323,13 @@ public override Type MakeByRefType()
             return SymbolType.FormCompoundType("&", this, 0)!;
         }
 
+        [System.Diagnostics.CodeAnalysis.RequiresDynamicCodeAttribute("The code for an array of the specified type might not be available.")]
```

Comment:
Add `using System.Diagnostics.CodeAnalysis;` at the top instead of the full namespace here?